Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenuTemplate.xaml

```diff
@@ -119,7 +119,6 @@
                                               ScrollViewer.VerticalScrollBarVisibility="Auto"
                                               SelectedIndex="{Binding SelectedIndex, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
                                               SelectedItem="{Binding SelectedItem, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
-                                              SelectionMode="Single"
```

Comment:
Why remove the selection mode?

Reply:
No idea, but having it there caused the two way binding on the selecteditem to be reset to null when the control was loaded. It feels like a platform bug as the SelectionMode by default is single.